### ISSUE TITLE
allow --force-device-scale-factor=1 as setting in seleniumbase

### ISSUE THREAD

**Issue by svenstehle** [seleniumbase/SeleniumBase#1294]
Hi, I want to run my tests on GitHub Actions CI and I am writing the tests locally on macos. The retina display that I am using consistently doubles `width` and `height`. 

I addressed this by crudely monkeypatching `get_new_driver` of `BaseCase`:

```python
from selenium import webdriver
from seleniumbase import BaseCase


class ComponentsTest(BaseCase):
    def get_new_driver(self, *args, **kwargs):
        """ This method overrides get_new_driver() from BaseCase. """
        options = webdriver.ChromeOptions()
        options.add_argument("--force-device-scale-factor=1")    # fix macos retina displays
        if self.headless:
            options.add_argument("--headless")
        driver = webdriver.Chrome(options=options)
        width = 1920
        height = 1080
        driver.set_window_size(width, height)
        print('Window size', driver.get_window_size())
        return driver
```

But this is a very crude solution and it broke on my second test case when I used `self.click`. How can I cleanly enable any google chrome browser option that I can pass to `selenium.webdriver` in that way, also work in `seleniumbase`?

I found `custom_settings.py` but I don't know if that is the right place?

Help appreciated, thank you :)

**Comment by svenstehle** [comment]
I figured it out for one argument: `--chromium-arg="force-device-scale-factor=1"` works as I want it to and fixes the `width` and `height` to the specified values in the `custom_settings.py`. Is this the only way?

And: what is the correct syntax if I want to provide more than just one additional argument? Something like:
`--chromium-arg="force-device-scale-factor=1 disable-gpu"` ?

**Comment by svenstehle** [comment]
@mdmintz  Thanks for the quick response! 

I tinkered with it some more and I can confirm that 

```bash
pytest tests --settings-file=tests/custom_settings.py --chromium-arg="force-device-scale-factor=1,headless"
```

works like you said :) Scaling factor is set AND the browser starts headless.

Is that already in the documentation somewhere? I could not find examples besides the comment next to the option: 

```bash
--chromium-arg=ARG # (Add a Chromium arg for Chrome/Edge, comma-separated.)
```

Would you be open to extend the documentation with an exact syntax example for this option?

**Comment by mdmintz** [comment]
Hi @svenstehle That's in the documentation: https://github.com/seleniumbase/SeleniumBase/blob/master/help_docs/customizing_test_runs.md

Click on "Options" near the top of the main ReadMe file.

<img width="666" alt="Screen Shot 2022-05-03 at 12 58 02 PM" src="https://user-images.githubusercontent.com/6788579/166502681-367079f2-d94d-4210-a69f-07d6201c771a.png">

And ``--headless`` already exists as a built-in SeleniumBase option.

```bash
pytest tests --headless --settings-file=tests/custom_settings.py --chromium-arg="force-device-scale-factor=1"
```

**Comment by svenstehle** [comment]
Maybe I misunderstand. I cannot find an example in the ReadMe like the one you showed me.

> Eg `pytest --chromium-arg="ARG1=1,ARG2=2"` (no spaces - use quotes around everything)

**Comment by mdmintz** [comment]
It's right here:

<img width="1086" alt="Screen Shot 2022-05-03 at 1 14 07 PM" src="https://user-images.githubusercontent.com/6788579/166504756-02fd9354-b23b-4542-b090-653cc8542679.png">

**Comment by svenstehle** [comment]
> It's right here:
> 
> <img alt="Screen Shot 2022-05-03 at 1 14 07 PM" width="1086" src="https://user-images.githubusercontent.com/6788579/166504756-02fd9354-b23b-4542-b090-653cc8542679.png">

I don't want to appear cheeky or rude here. I think the helpful and concise example that you have given in this issue thread could improve the documentation right there. 

Suggestion, from your example:

```bash
--chromium-arg="ARG1=1,ARG2=2" # (Add a Chromium arg for Chrome/Edge, comma-separated, no spaces.)
```

I am open to create a PR for this.

**Comment by mdmintz** [comment]
@svenstehle I just made some changes:

https://github.com/seleniumbase/SeleniumBase/commit/30eafb00d2023e2c4833e34e4636c8147554963e

```bash
--chromium-arg="ARG=N,ARG2"  # (Set Chromium args, ","-separated, no spaces.)
--firefox-arg="ARG=N,ARG2"  # (Set Firefox args, comma-separated, no spaces.)
```

**Comment by svenstehle** [comment]
Awesome, that looks good! I think that will help a few newbies like me a lot. Thanks :)

**Comment by nakodavati** [comment]
if i pass args like this is this correct ?  --chromium-arg="--no-sandbox,--disable-dev-shm-usage" will it take these

**Comment by mdmintz** [comment]
@nakodavati Yes. (Note that both those args are already set by SeleniumBase if running on Linux.)

**Comment by nakodavati** [comment]
@mdmintz  how can i make sure it is  already set by seleniumBase , inside container while executing tests i am proving these options as command line argument

**Comment by mdmintz** [comment]
@nakodavati In a test, use ``self.driver.options.arguments`` to see the options used when the browser was launched.
